content/api/commands/clearcookies.md

```diff
@@ -40,8 +40,7 @@ Pass in an options object to change the default behavior of `cy.clearCookies()`.
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>`cy.clearCookies()` yields `null`.</li><li>`cy.clearCookies()` cannot
-be chained further.</li></List>
```

Comment:
I think it's valuable to callout it cannot be chained further.

Reply:
We chain off commands that return `null` all the time in our own tests.

```
it.only('works', () => {
  cy.visit('/fixtures/dom.html')
    .end()
    .get('button')
    .end()
    .clearCookies('foo')
    .get('form')
})
```

It's always worked this way, our docs were just plain wrong. :woman_shrugging: